rotkehlchen/chain/ethereum/modules/yearn/utils.py

```diff
@@ -138,6 +117,10 @@ def query_yearn_vaults(db: 'DBHandler') -> None:
             log.error(f'Could not identify the yearn vault type for {vault}. Skipping...')
             continue
 
+        if (version := vault.get('version')) is not None and version.startswith('3.'):
```

Comment:
this should go away in develop merge no?

Reply:
yes